pytissueoptics/scene/tree/node.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, parent: 'Node' = None, polygons: List[Polygon] = None, bbox:
         self._polygons = polygons
         self._bbox = bbox
         self._depth = depth
+        self._visited = False
 
     @property
     def parent(self):
```

Comment:
Not used.

Reply:
details needed.

Reply:
public node.parent is never called in the code base

Reply:
Resolved with 4408122614e7a8e265f5b5f95ee0644c2040a1cd